src/types/components.d.ts

```diff
@@ -120,17 +120,25 @@ export interface ImageProps {
    */
   lazy?: boolean;
   /**
```

Comment:
The optional "gradient" property is missing in this type which leads to a compilation error in the Image component.
```suggestion
  /**
  * Should the gradient overlay be applied to the image?
  * 
  * Default value is: **false**
  */
  gradient?: boolean;
  /**
```